deploy/crds/crd-certificates.yaml

```diff
@@ -207,6 +207,20 @@ spec:
                 secretName:
                   description: SecretName is the name of the secret resource that will be automatically created and managed by this Certificate resource. It will be populated with a private key and certificate, signed by the denoted issuer.
                   type: string
+                secretTemplate:
+                  description: SecretTemplate defines annotations and labels to be propagated to the Kubernetes Secret when it is created or updated.
+                  type: object
+                  properties:
+                    annotations:
+                      description: Annotations is a key value map to be copied to the target Kubernetes Secret.
+                      type: object
+                      additionalProperties:
+                        type: string
+                    labels:
+                      description: Labels is a key value map to be copied to the target Kubernetes Secret.
+                      type: object
+                      additionalProperties:
+                        type: string
```

Comment:
```
kubectl explain certificate.spec.secretTemplate
KIND:     Certificate
VERSION:  cert-manager.io/v1

RESOURCE: secretTemplate <Object>

DESCRIPTION:
     SecretTemplate defines annotations and labels to be propagated to the
     Kubernetes Secret when it is created or updated.

FIELDS:
   annotations  <map[string]string>
     Annotations is a key value map to be copied to the target Kubernetes
     Secret.

   labels       <map[string]string>
     Labels is a key value map to be copied to the target Kubernetes Secret.

```

Please add a note to explain that labels and annotations are not yet removed from the Secret when they are removed from the template and link to a followup issue in GitHub.

Reply:
Note added.